cmd.js

```diff
@@ -92,8 +93,8 @@ start().catch(err => {
 async function start () {
   const brittle = require('./')
 
-  if (bail || solo) {
-    brittle.configure({ bail, solo })
+  if (bail || solo || timeout) {
+    brittle.configure({ bail, solo, timeout: +timeout })
```

Comment:
this overwrites the default value of timeout due to JS, you need to do:

```
timeout: timeout ? Number(timeout) : undefined
```

Reply:
updated